ui/app/components/job-search-box.hbs

```diff
@@ -0,0 +1,20 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+~}}
+
+<@S.TextInput
+  @type="search"
+  @value={{@searchText}}
+  aria-label="Job Search"
+  placeholder="Name contains myJob"
```

Comment:
hinting that this handles filter expressions and not just strings

Reply:
Noting this as the moment where I learned about https://github.com/hashicorp/js-bexpr